cpp/FileTransferAgent.cpp

```diff
@@ -19,7 +19,7 @@
 #include "EncryptionProvider.hpp"
 #include "logger/SFLogger.hpp"
 #include "snowflake/platform.h"
-#include "snowflake/Simba_CRTFunctionSafe.h"
+#include "snowflake/sb_CRTFunctionSafe.h"
```

Comment:
why do we rename this file? isn't it kind of our public API?

Reply:
remove Simba from the file name. We can keep the existing name though please let me know which one you prefer.

Reply:
reverted. Will handle that with a separated issue and PR